src/DelayTagged.h

```diff
@@ -30,10 +31,10 @@ class DelayTaggedBucket : public RefCountable
     typedef RefCount<DelayTaggedBucket> Pointer;
 
     void stats(StoreEntry *)const;
-    DelayTaggedBucket(String &aTag);
+    DelayTaggedBucket(const SBuf &aTag);
     ~DelayTaggedBucket() override;
     DelayBucket theBucket;
-    String tag;
+    SBuf tag;
```

Comment:
From compiler point of view, we can make this (and possibly the other new SBuf field) `const` AFAICT, but I did _not_ do it because I am not sure whether this tag is meant to be preserved throughout DelayTaggedBucket lifetime and because one could consider such a change out of this PR scope. @kinkie, your call.

Reply:
It can work as const. I'll request a re-review soon

Reply:
... and please check that other new SBuf field as well if you decide to go down this path.

Reply:
CompositeSelectionDetails seems to be never modified after construction.
Moving the data setter in the constructor, I could make both always-present fields const